internal/pkg/service/stream/dependencies/dependencies.go

```diff
@@ -46,7 +46,6 @@ type ServiceScope interface {
 	dependencies.PublicScope
 	dependencies.EtcdClientScope
 	dependencies.TaskScope
-	Config() config.Config
 }
```

Comment:
Removed top-level config from the dependencies container.

It is anti-pattern (with such large app) to pass the whole configuration when only part of it is needed. It leads to cyclic dependencies between packages, since the top-level configuration refers to almost every package.

Reply:
Instead, when creating individual services, only the configuration that is needed is passed.